src/Microsoft.Health.Fhir.Shared.Api/Controllers/FhirController.cs

```diff
@@ -512,5 +513,20 @@ public async Task<IActionResult> Versions()
 
             return new OperationVersionsResult(response, HttpStatusCode.OK);
         }
+
+        /// <summary>
+        /// Handles batch and transaction requests
+        /// </summary>
+        /// <param name="bundle">The bundle being posted</param>
+        [HttpPost]
+        [Route("", Name = RouteNames.PostBundle)]
+        [AuditEventType(AuditEventSubType.Batch)] // TODO: Our current auditing implementation only allows one audit event type attribute even though this action handles two.
+        [Authorize(PolicyNames.WritePolicy)]
```

Comment:
Batch can also do read right?

Should we have batch as a separate policy?

Reply:
My original thought was that `write` implicitly allowed `read`, but this doesn't seem to be the way it's currently implemented. We have this exact same issue today with `ConditionalCreate` and `ConditionalUpdate`. Let me give this some more thought.

Reply:
I ended up just adding `write` and `read` policies.